test/e2e/util.go

```diff
@@ -108,7 +108,7 @@ func waitSizeReachedWithFilter(t *testing.T, f *framework.Framework, clusterName
 				names = append(names, pod.Name)
 			}
 		}
-		t.Logf("waiting size (%d), etcd pods: %v", size, names)
+		t.Logf("%v waiting size (%d), etcd pods: %v", time.Now(), size, names)
```

Comment:
```go

func logWithTimeStamp(t *testing.T, format string, args ...interface{}) {
    t.Logf("%v " + format, time.Now(), args...)
}
```

this might need some little tweak.